projects/power_distribution/inc/smoke_uv_cutoff.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+// A smoke test mode for the UV detector
+void smoke_uv_cutoff_perform(void);
```

Comment:
For the last step in hooking your smoketest into the system, you'll need to add a line to the `s_smoke_tests` array in smoketests_pd.c. The purpose of that array is to map the `SmokeTestMode` enum values from smoketests_pd.h to functions like this, which are the actual functions called when somebody sets `PD_SMOKE_TEST` to a certain value in smoketests_pd.h.

Currently `SMOKE_TEST_CURRENT_MEASUREMENT` is mapped to the `smoke_current_measurement_perform`  function from smoke_current_measurement.h; you'll need to additionally map `SMOKE_TEST_UV_CUTOFF` to `smoke_uv_cutoff_perform` using the same syntax (it's called "designated inits" and it's a GNU extension to C, see https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html).